recipe/node.go

```diff
@@ -85,3 +86,13 @@ func (node RecipeNode) toSinks() (sinks []PluginRecipe, err error) {
 	}
 	return
 }
+
+// toSupportMultipleSrcTags helps to support both tags `name` and `type` for source
+// till `type` tag gets deprecated
+func (node RecipeNode) toSupportMultipleSrcTags() string {
```

Comment:
this only handles `type` when linting. But what about other commands like `run`? Will source be able to use `name` and `type` interchangeably?

Reply:
Yes, its working fine with other cmd too, have checked it.